core/lib/types/src/block.rs

```diff
@@ -62,6 +62,12 @@ pub struct L1BatchHeader {
     pub base_system_contracts_hashes: BaseSystemContractsHashes,
     /// Version of protocol used for the L1 batch.
     pub protocol_version: Option<ProtocolVersionId>,
+    /// The commitment to the final events queue state after the batch is committed.
+    /// Practically, it is a commitment to all events that happened on L2 during the batch execution.
```

Comment:
love the comments - thank you.

Reply:
I copied them from `boojum-integration` branch 😅